packages/hardhat/hardhat.config.ts

```diff
@@ -10,6 +17,40 @@ const config: HardhatUserConfig = {
       default: 0,
     },
   },
+  networks: {
+    arbiturum: {
+      url: `https://arb-mainnet.g.alchemy.com/v2/${defaultApiKey}`,
+      accounts: [deployerPrivateKey]
+    },
+    arbiturumTestnet: {
+      url: `https://arb-rinkeby.g.alchemy.com/v2/${defaultApiKey}`,
```

Comment:
Arbitrum rinkeby was deprecated after the merge. Let's switch to arbitrumGoerli. We can also change the name: arbitrumTestnet => arbitrumGoerli

Reply:
fixed here https://github.com/scaffold-eth/se-2/pull/24/commits/634dc5c58f8a6ea6f51bf6bc7e7389ad11edd734